internal/shared/api/handlers/handler/handler.go

```diff
@@ -17,12 +18,22 @@ import (
 )
 
 type Handler struct {
-	Config  *config.Config
-	Service service.SharedServiceProvider
+	Config    *config.Config
+	Service   service.SharedServiceProvider
+	bbnClient *bbnclient.BBNClient
 }
 
 func New(config *config.Config, service service.SharedServiceProvider) (*Handler, error) {
-	return &Handler{Config: config, Service: service}, nil
+	var bbnClient *bbnclient.BBNClient
```

Comment:
the reason why I didn't move it into shared service is because we will rewrite the way our code structured anyway and didn't want to complicate things for now